deployer/cluster.py

```diff
@@ -73,43 +73,43 @@ def deploy_support(self):
         cert_manager_url = "https://charts.jetstack.io"
         cert_manager_version = "v1.3.1"
 
-        # print_colour("Adding cert-manager chart repo...")
-        # subprocess.check_call(
-        #     [
-        #         "helm",
-        #         "repo",
-        #         "add",
-        #         "jetstack",
-        #         cert_manager_url,
-        #     ]
-        # )
-
-        # print_colour("Updating cert-manager chart repo...")
-        # subprocess.check_call(
-        #     [
-        #         "helm",
-        #         "repo",
-        #         "update",
-        #     ]
-        # )
-
-        # print_colour("Provisioning cert-manager...")
-        # subprocess.check_call(
-        #     [
-        #         "helm",
-        #         "upgrade",
-        #         "--install",
-        #         "--create-namespace",
-        #         "--namespace=cert-manager",
-        #         "cert-manager",
-        #         "jetstack/cert-manager",
-        #         f"--version={cert_manager_version}",
-        #         "--set=installCRDs=true",
-        #     ]
-        # )
-        # print_colour("Done!")
-
-        # print_colour("Provisioning support charts...")
+        print_colour("Adding cert-manager chart repo...")
+        subprocess.check_call(
+            [
+                "helm",
+                "repo",
+                "add",
+                "jetstack",
+                cert_manager_url,
+            ]
+        )
+
+        print_colour("Updating cert-manager chart repo...")
+        subprocess.check_call(
+            [
+                "helm",
+                "repo",
+                "update",
+            ]
+        )
+
+        print_colour("Provisioning cert-manager...")
+        subprocess.check_call(
+            [
+                "helm",
+                "upgrade",
+                "--install",
+                "--create-namespace",
+                "--namespace=cert-manager",
+                "cert-manager",
+                "jetstack/cert-manager",
+                f"--version={cert_manager_version}",
+                "--set=installCRDs=true",
```

Comment:
I'll open a separate PR with this change after this is merged to avoid merge conflicts.

This is an optimization to not need to: a) add a named helm chart repo, b) updating all helm chart repos, c) referencing a helm chart repo via named helm chart repo. Instead one would simply name the helm chart and declare the repo, which is fetched just in time of the upgrade.

```suggestion
        print_colour("Provisioning cert-manager...")
        subprocess.check_call(
            [
                "helm",
                "upgrade",
                "cert-manager",  # given release name (aka. installation name)
                "cert-manager",  # helm chart to install
                f"--repo={cert_manager_url}",
                "--install",
                "--create-namespace",
                "--namespace=cert-manager",
                f"--version={cert_manager_version}",
                "--set=installCRDs=true",
```

Reply:
Perfect! Would definitely <3 a new PR!

Reply:
Retained memory of doing this in #1100